test/integration/nfs-shared-volumes.test.js

```diff
@@ -172,7 +174,7 @@ test('setup', function (tt) {
 });
 
 test('nfs shared volumes', function (tt) {
-    var mountPoint;
+    var mountPoint = '/mnt';
```

Comment:
I'd personally use a constant on these cases. Not a reason to do not approve the PR though :-)